src/test/rustdoc/type-layout.rs

```diff
@@ -52,3 +52,12 @@ pub struct Unsized([u8]);
 
 // @!has type_layout/trait.MyTrait.html 'Size: '
 pub trait MyTrait {}
+
+// @has type_layout/enum.Variants.html 'Size: '
+// @has - '2 bytes'
+// @has - '<code>A</code>: 0 bytes'
+// @has - '<code>B</code>: 1 byte'
+pub enum Variants {
+    A,
+    B(u8),
+}
```

Comment:
Can you add this as a test case?

```rust
// @has type_layout/enum.WithNiche.html 'Size: '
// @has - '16 bytes'
// @has - '<code>None</code>: 0 bytes'
// @has - '<code>Some</code>: 16 bytes'
pub enum WithNiche {
    None,
    Some(&'a u32, u64),
}
```

Reply:
Done. @rustbot label -S-waiting-on-author S-waiting-on-review